internal/receiver/databricksreceiver/config.go

```diff
@@ -0,0 +1,63 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package databricksreceiver
+
+import (
+	"errors"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+
+	"github.com/signalfx/splunk-otel-collector/internal/receiver/databricksreceiver/internal/metadata"
+)
+
+func createDefaultConfig() component.Config {
+	scs := scraperhelper.NewDefaultScraperControllerSettings(typeStr)
+	// set the default collection interval to 30 seconds which is half of the
+	// lowest job frequency of 1 minute
+	scs.CollectionInterval = time.Second * 30
+	return &Config{
+		MaxResults:                25, // 25 is the max the API supports
```

Comment:
should the max be validated or is it better to marshal fetch failures at run time so we don't have to track it?

Reply:
Validation added.